packages/linux_dpms/lib/src/linux_dpms_web.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:linux_dpms/src/dpms_power_state.dart';
+
+class DpmsLinux {
```

Comment:
If you allow `DPMS.instance` returning null, you don't need to provide a DPMS implementation for web, just make it return null there. But that's your call